Formula/cava.rb

```diff
@@ -0,0 +1,57 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool"  => :build
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # change ncursesw to ncurses
+    inreplace "configure.ac", "ncursesw", "ncurses"
+    # force autogen.sh to look for and use our glibtoolize
+    inreplace "autogen.sh", "libtoolize", "glibtoolize"
+
+    # to be remove with versions greater 0.7.4:
+    # correct Makefile.am with hardcoded libpath
+    inreplace "Makefile.am", "/usr/local", HOMEBREW_PREFIX unless build.head?
+
+    system "./autogen.sh"
+    system "./configure", *std_configure_args, "--disable-silent-rules"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    cava_config = (testpath/"cava.conf")
+    cava_stdout = (testpath/"cava_stdout.log")
+
+    (cava_config).write <<~EOS
+      [general]
+      bars = 2
+      sleep_timer = 1
+
+      [input]
+      method = fifo
+      source = /dev/zero
+
+      [output]
+      method = raw
+      data_format = ascii
+    EOS
+
+    pid = spawn("#{bin}/cava -p #{cava_config}", [:out, :err] => cava_stdout.to_s)
```

Comment:
```suggestion
    pid = spawn(bin/"cava", "-p", "cava_config", [:out, :err] => cava_stdout.to_s)
```
Please don't pass this as one string. See my previous suggestion.

Reply:
Sorry.